lib/c_glib/test/owncloud.level2crm.pem

```diff
@@ -0,0 +1,59 @@
+X8JTar9zbANnJIR89fSXp4wQUbTKt7xbBezv8c/Cel6dlTSC8Dvc50ptksh5yue2
+SEuglFDtf7ru8EidROMgvUw4VAjusZXVirPKjv2rqWamgRsWizq9nkqJ71HjFweo
+FYQRIuZ6qSDl3x7ZDV22kZth2LxFHRnv1E/SHbsvs0iEoKr9hF6xkdxyfHmlyd02
+oJOdlxEfHSREtuFDbDP/CcQy5oVbdM2EfQIDAQABo4IBWjCCAVYwCQYDVR0TBAIw
+ADARBglghkgBhvhCAQEEBAMCBPAwCwYDVR0PBAQDAgXgMCUGCWCGSAGG+EIBDQQY
+FhZMZXZlbDIgQ1JNIENlcnRpZmljYXRlMB0GA1UdDgQWBBTa17QOLz4WqHv0J6LP
+KVVPsX721TCB4gYDVR0jBIHaMIHXgBTRF2IG48O6cMjz44Pwk5GbhUWX3aGBs6SB
+sDCBrTELMAkGA1UEBhMCRVMxDzANBgNVBAgTBk1hZHJpZDEVMBMGA1UEBxMMVG9y
+cmVsb2RvbmVzMSswKQYDVQQKEyJMZXZlbDIgU29sdWNpb25lcyBNYXJrZXRpbmcg
+MzYwIFNMMREwDwYDVQQLEwhTaXN0ZW1hczEPMA0GA1UEAxMGTGV2ZWwyMSUwIwYJ
+KoZIhvcNAQkBFhZnYWd1aWxhckBsZXZlbDJjcm0uY29tggkAqMKCeCp/2+wwDQYJ
+KoZIhvcNAQELBQADggEBACy1UZR674b3yxEYnFPorhbD4CS65AfgLEObpvwbGvF8
+gLxJkxGhcsKEoxiBynnazdPzub8I9e0ZHsPjAyPupLrBeuUIBthmzdW3gUN9zZp1
+mRIurbiT3wlYCxl/cOa7MV7bCZSSqsC0WF9CLicRPFgyc6MDdRjsMufx+JkIdJZa
+31jBweG/JzYm9pxdJFfnFcBEM+Kv3BcozoAAHcLHvv4cI4j5SpdzdCEgpgd6/LtI
+i/grRayPlbQnqUrEOHLAuhAH+RflRTIFBGRRukLa7yqdY9J+qHDPLkxnLEmtyu4u
+6GoZTYFtKnVZfubr+gku9Z8r2gF375InjOAifZH3ITE=
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+MIIFBDCCA+ygAwIBAgIJAKjCgngqf9vsMA0GCSqGSIb3DQEBCwUAMIGtMQswCQYD
+VQQGEwJFUzEPMA0GA1UECBMGTWFkcmlkMRUwEwYDVQQHEwxUb3JyZWxvZG9uZXMx
+KzApBgNVBAoTIkxldmVsMiBTb2x1Y2lvbmVzIE1hcmtldGluZyAzNjAgU0wxETAP
+BgNVBAsTCFNpc3RlbWFzMQ8wDQYDVQQDEwZMZXZlbDIxJTAjBgkqhkiG9w0BCQEW
+FmdhZ3VpbGFyQGxldmVsMmNybS5jb20wHhcNMTUwMTAxMTQyMDQ1WhcNMjQxMjI5
+MTQyMDQ1WjCBrTELMAkGA1UEBhMCRVMxDzANBgNVBAgTBk1hZHJpZDEVMBMGA1UE
+BxMMVG9ycmVsb2RvbmVzMSswKQYDVQQKEyJMZXZlbDIgU29sdWNpb25lcyBNYXJr
+ZXRpbmcgMzYwIFNMMREwDwYDVQQLEwhTaXN0ZW1hczEPMA0GA1UEAxMGTGV2ZWwy
+MSUwIwYJKoZIhvcNAQkBFhZnYWd1aWxhckBsZXZlbDJjcm0uY29tMIIBIjANBgkq
+hkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA2RJKzAzO8P0GT3FcnFMK3X9t4JE+mHMP
+aPX5miBLaKH6rXclrfjYfzK3OESY5DhgZJ+Qzx5GyNl+hsSXpiIlNbWAEeJCIELG
+OEz51zR0M2rQTC06UrWTtn2COQo2m91lv+Mwum+LSgqUPJ18dyKsmOFSVCsLOK+s
+xZY7F5AF2+99kr1EFv0dcxS1BTRKOAQiMkuZEnFU95zXmm+U+vzenwX2V8XtTLip
+e6+MezZTehvSZhbafKwFnw+vc36VGwrfhc+lH/1pc5etp4qcD9L0WvOhTmBUwWl/
+2hOU0XF5UYOkfsS1saot/WIsH6731Q/slYFCLXSrl+8R4+RxR9fozwIDAQABo4IB
+IzCCAR8wHQYDVR0OBBYEFNEXYgbjw7pwyPPjg/CTkZuFRZfdMIHiBgNVHSMEgdow
+gdeAFNEXYgbjw7pwyPPjg/CTkZuFRZfdoYGzpIGwMIGtMQswCQYDVQQGEwJFUzEP
+MA0GA1UECBMGTWFkcmlkMRUwEwYDVQQHEwxUb3JyZWxvZG9uZXMxKzApBgNVBAoT
+IkxldmVsMiBTb2x1Y2lvbmVzIE1hcmtldGluZyAzNjAgU0wxETAPBgNVBAsTCFNp
+c3RlbWFzMQ8wDQYDVQQDEwZMZXZlbDIxJTAjBgkqhkiG9w0BCQEWFmdhZ3VpbGFy
+QGxldmVsMmNybS5jb22CCQCowoJ4Kn/b7DAMBgNVHRMEBTADAQH/MAsGA1UdDwQE
+AwIBBjANBgkqhkiG9w0BAQsFAAOCAQEAh2kPzv8jFNQbCrAD770m8T4W/KMhxMOv
+Uhm/wSv7/g3Ir1pZ9LgXAWJx1qLQdCcOikBparkUpHIrqDU+wBL6o+CFNWp7WR6g
+bft627bL9R3z3fTQHIchjPHQ7S4ID/STWfTqWDJQESBnTp1KTuJ8sMlvACMWZSUc
+jZOAQIodNiUUNPkfreurPb9KH/nArwV/yTDoZMhiqoqNwOIY/0sxUJaNUAX3pgDd
+FsGMBn4MiyGAw4XaK9LJBh+6DLH5D2Y7h0hXv+AUP3SSC+w5XA/D+SHcAbjtPzMW
+cdq9jr1Gu0KO3PSGIAUpI1o+XQeLqgyieFcVF/FlD31MDC152DifNA==
+-----END CERTIFICATE-----
```

Comment:
We could make use of cert files under `test/keys` directory instead of adding new one.
That way we can easily add c_glib ssl to the cross test too.